time_tracker_api/activities/activities_model.py

```diff
@@ -7,13 +7,17 @@ class ActivityDao(CRUDDao):
 
 def create_dao() -> ActivityDao:
     from time_tracker_api.sql_repository import db
-    from time_tracker_api.sql_repository import SQLCRUDDao, AuditedSQLModel
+    from time_tracker_api.sql_repository import SQLCRUDDao
+    from sqlalchemy_utils import UUIDType
+    import uuid
 
-    class ActivitySQLModel(db.Model, AuditedSQLModel):
+    class ActivitySQLModel(db.Model):
         __tablename__ = 'activity'
-        id = db.Column(db.Integer, primary_key=True)
+        id = db.Column(UUIDType(binary=False), primary_key=True, default=uuid.uuid4)
         name = db.Column(db.String(50), unique=True, nullable=False)
         description = db.Column(db.String(250), unique=False, nullable=False)
+        deleted = db.Column(db.String, default=None)
+        tenant_id = db.Column(db.String, nullable=False)
```

Comment:
Let's change all id fields that are supposed to be `UUID` to the right type. The use of `String` was actually a workaround.